### ISSUE TITLE
Unexpectedly found nil while implicitly unwrapping an Optional value

### ISSUE THREAD

**Issue by ian-sayles** [twostraws/CodeScanner#77]
On a new device app keeps crashing when asked to give permission to access the camera.
It crashes with the following error below, this happens before the 'OK' button is pressed, if you press the OK button, and start the app again it works ok.

CodeScanner/ScannerViewController.swift:178: Fatal error: Unexpectedly found nil while implicitly unwrapping an Optional value
2022-08-12 09:30:14.185667+0100 [405:12472] CodeScanner/ScannerViewController.swift:178: Fatal error: Unexpectedly found nil while implicitly unwrapping an Optional value

if previewLayer == nil {
previewLayer = AVCaptureVideoPreviewLayer(session: captureSession)
}

The version of Codescanner 2.1.2 and iOS 15.6 and happens on iPad's and iPhone's

**Comment by NathanFallet** [comment]
Same error in production as well
<img width="870" alt="image" src="https://user-images.githubusercontent.com/30439790/185784018-87f894e6-5c76-4111-b455-de6683aaa183.png">
<img width="678" alt="image" src="https://user-images.githubusercontent.com/30439790/185784032-05017a3c-430e-47b8-ab0e-9f448935d2f2.png">

**Comment by NathanFallet** [comment]
Fixed by #73, just need to wait for a release

**Comment by MartinaRogach** [comment]
> Fixed by #73, just need to wait for a release

when do you release this fix?

**Comment by NathanFallet** [comment]
> > Fixed by #73, just need to wait for a release
> 
> 
> 
> when do you release this fix?

We're waiting for @twostraws to tag a new version

**Comment by twostraws** [comment]
This has just been done. Sorry for the delay – I'm still on vacation!

**Comment by NathanFallet** [comment]
Thanks for merging! Have you seen my email about this repository? (Sent on Aug 22 to find it easily)

**Comment by twostraws** [comment]
@NathanFallet No; I’m on vacation.

**Comment by NathanFallet** [comment]
@twostraws Yes I know, and I saw you don’t have many time to maintain this repo.
That’s why I would like to help you and become a maintainer (“collaborator” on GitHub) on the project. Like I already do with SQLite.swift and CryptoSwift, I can help managing issues (sort, reply, or close if needed), pull requests (review and merge if approved), and releases (create a tag when needed).
I already made some contributions on it, and I would love to help more! Tell me what you think about it.
